Helper/Webhook.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Klaviyo\Reclaim\Helper;
+
+use Exception;
+use \Klaviyo\Reclaim\Helper\ScopeSetting;
+
+class Webhook extends \Magento\Framework\App\Helper\AbstractHelper
+{
+    const USER_AGENT = 'Klaviyo/MagentoTwo/Webhook';
+//    const WEBHOOK_URL = 'https://www.klaviyo.com/api/webhook/integration/magento_two';
+    const WEBHOOK_URL = 'https://rem-klaviyo.ngrok.io/test';
+
+    /**
+     * Klaviyo scope setting helper
+     * @var \Klaviyo\Reclaim\Helper\ScopeSetting $klaviyoScopeSetting
+     */
+    protected $_klaviyoScopeSetting;
+
+    public function __construct(
+        ScopeSetting $klaviyoScopeSetting
```

Comment:
is there a reason for this being on its own line?

Reply:
yea, just how its styled